services/community-site/src/pages/Staking/NodeProfilePage.tsx

```diff
@@ -331,54 +327,150 @@ const NodeProfilePage = () => {
               </div>
             </div>
           </div>
-          {delegations.length !== 0 && (
-            <>
-              <hr className="nodeInfoDivider" />
+          <hr className="nodeInfoDivider" />
+          <div className="nodeTypes">
+            <div className="nodeTitleContainer">
+              <NodeIcon />
+              <Text label="Stats" variant="h6" color="primary" className="box-title" />
+              <Button
+                size="small"
+                className={clsx('nodeTypeTab', detailType === ViewType.DELEGATIONS && 'active')}
+                label="Delegations"
+                variant="contained"
+                onClick={() => {
+                  setDetailType(ViewType.DELEGATIONS);
+                }}
+              />
+              <Button
+                size="small"
+                className={clsx(
+                  'nodeTypeTab',
+                  detailType === ViewType.COMMISSION_CHANGES && 'active',
+                )}
+                label="Commissions"
```

Comment:
Should be named "Commission Changes"

Reply:
I left it as "Commissions" cuz it looks kinda off if written out
![image](https://github.com/Taraxa-project/taraxa-platform/assets/100562086/eac48746-1592-4faf-b27b-a45ef10a44bd)